portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+/** The structure for comments */
+public final class Comment {
+
+  private final long id;
+  private final String comment;
+  private final long timestamp;
+  private final Double sentiment;
+
+  // Constructor for the Comment Class, it takes in a long ID and timestap and a String comment
```

Comment:
Consider updating this comment?

I'd also consider not listing the parameters in this comment, since that doesn't add much info that you can't get from the code.